Documentation/data-collection.md

```diff
@@ -257,6 +257,12 @@ data:
     # cluster:vmi_request_cpu_cores:sum is the total number of CPU cores requested by pods of VMIs.
     - '{__name__="cluster:vmi_request_cpu_cores:sum"}'
     #
+    # owners: (https://github.com/kubev2v/forklift)
+    #
+    # cluster:mtv_migrations_status_total is the total number of VM migrations running on the cluster,
+    # labeled with {status}, {provider}, {mode}, and {target}.
+    - '{__name__="mtv_migrations_status_total"}'
```

Comment:
the metrics name `mtv_migrations_status_total` is not the same as `cluster:mtv_migrations_status_total` in
`cluster:mtv_migrations_status_total is the total number of VM migrations running on the cluster,`
maybe should be `cluster:mtv_migrations_status_total:sum` as @simonpasquier mentioned in https://github.com/openshift/cluster-monitoring-operator/pull/2461#pullrequestreview-2292267082